packages/bot-web-ui/src/components/run-panel.jsx

```diff
@@ -98,30 +98,45 @@ const drawerFooter = ({
     );
 };
 
-class RunPanel extends React.PureComponent {
+class RunPanel extends React.Component {
     componentDidMount() {
         this.props.onMount();
     }
 
     componentWillUnmount() {
         this.props.onUnmount();
     }
-
+    onToggleModal = () => {
+        const { initialSelfExclustion, setLimitations } = this.props;
+        initialSelfExclustion();
```

Comment:
Do we have to call this when toggling the modal? On stopping the bot + on first initialisation the values should be in their initial state, right?

Reply:
> Do we have to call this when toggling the modal? On stopping the bot + on first initialisation the values should be in their initial state, right?

Yes, we need this. in some cases, Modal doesn't close when we set values